app/views/spree/admin/reports/_table.html.haml

```diff
@@ -0,0 +1,23 @@
+- column_partials ||= {}
+- if render_content?
+  %table.report__table{id: id}
+    %thead
+      %tr
+        - @header.each do |heading|
+          %th= heading
+    %tbody
+      - @table.each do |row|
+        %tr
+          - row.each_with_index do |cell_value, column_index|
+            %td
+              - partial = column_partials[column_index]
+              - if partial
+                = render partial, value: cell_value
+              - else
+                = cell_value
+      - if @table.empty?
+        %tr
+          %td{colspan: @header.count}= t(:none)
+- else
+  %p.report__message
+    = t(".select_and_search")
```

Comment:
Do we have a strategy for implementing BEM? Should we just be updating things as we touch them? Should we write up a proposal about how to reimplement our existing styling so that devs know how to approach this when they have to deal with it?

Reply:
At the moment, we don't have a naming convention. So using BEM is as good as any other naming. Paul suggested using BEM years ago and I think it's a really good idea. Should we discuss that on our next delivery train catchup? If we decide to go with it, we should document it, yes.

Reply:
Yes, it'd be great to discuss about it but perhaps in a new dev catchup as we agreed on the delivery train catchup?